### PR TITLE
RN for audit logging for oc command

### PR DESCRIPTION
**Pull request by codyhoag**

### CODE REVIEW

**Comment by codyhoag** [comment]
@kasturinarra can you please review this? I had a hard time tracking down any additional info for this, so I'm not sure if it's correct. Thanks!

**Comment by kasturinarra** [comment]
@codyhoag can we add something as below ? Run oc command with -- /usr/bin/gather_audit_logs to be able to collect the audit logs.
 
Not for documenting, but for your reference,  below is the help example just pasing for you
Examples:
  # gather information using the default plug-in image and command, writing into ./must-gather.local.<rand>
  oc adm must-gather
  
  # gather information with a specific local folder to copy to
  oc adm must-gather --dest-dir=/local/directory
  
  # gather audit information
  oc adm must-gather -- /usr/bin/gather_audit_logs
  
  # gather information using multiple plug-in images
  oc adm must-gather --image=quay.io/kubevirt/must-gather --image=quay.io/openshift/origin-must-gather
  
  # gather information using a specific image stream plug-in
  oc adm must-gather --image-stream=openshift/must-gather:latest
  
  # gather information using a specific image, command, and pod-dir
  oc adm must-gather --image=my/image:tag --source-dir=/pod/directory -- myspecial-command.sh

**Comment by kasturinarra** [comment]
@soltysh could you please help review as well ? Thanks !!

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -1257,6 +1257,12 @@ default.
 
 The default Jenkins Node.js agent has been upgraded to Node.js version 12.
 
+[discrete]
+[id="ocp-4-6-audit-logs-not-gathered-by-default"]
+==== Audit logs not gathered by default for `oc adm must-gather` command
+
+The `oc adm must-gather` command no longer collects audit logs by default. You must include the `--audit-logs` parameter in the `oc` command to gather audit logs.
```
Comment (soltysh) [inline release_notes/ocp-4-6-release-notes.adoc:1264]: You should use `oc adm must-gather -- /usr/bin/gather_audit_logs` command to gather audit logs.

**Review by soltysh — CHANGES_REQUESTED** [review]

**Comment by codyhoag** [comment]
@soltysh can you please confirm my edits? Thanks!

**Review by soltysh — APPROVED** [review]
/lgtm

**Comment by bmcelvee** [comment]
LGTM!

**Review by bmcelvee — APPROVED** [review]

**Pull request merged**